Vagrantfile

```diff
@@ -33,7 +33,8 @@ Vagrant.configure("2") do |config|
   config.vm.network :forwarded_port, guest: 8983, host: 18983
   config.vm.network :forwarded_port, guest: 3306, host: 13306
   config.vm.network :forwarded_port, guest: 1080, host: 1080
-  
+  config.vm.network :forwarded_port, guest: 3000, host: 8080
```

Comment:
I have it proxying 8080 because that in turn is forwarding to 8081 (in my setup, anyway.)